api/qsystem.py

```diff
@@ -53,6 +55,16 @@
 from app.patches.flask_oidc_patched import OpenIDConnect
 oidc = OpenIDConnect(application)
 
+from app import admin
+
+console = Admin(application, name='Admin Console', template_mode='bootstrap3')
```

Comment:
Can you give this a more verbose name, maybe like `admin_console`, or something?